bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -156,6 +156,11 @@ type ClusterConfiguration struct {
 	// The cluster name
 	// +optional
 	ClusterName string `json:"clusterName,omitempty"`
+
+	// CustomHostContainers contains the information of any additional images
+	// that we will deploy as host containers in the CPIs
+	// +optional
+	CustomHostContainers []BottlerocketHostContainer `json:"customHostContainers,omitempty"`
```

Comment:
If this can only be of type `BottlerocketHostContainer`, then we should probably call it `BottlerocketCustomHostContainers` right?

Reply:
Will do!